otter/test/convergence/test_service.py

```diff
@@ -104,24 +108,33 @@ def test_buckets_acquired(self):
         When buckets are allocated, the result of converge_all_groups is
         performed.
         """
-        def converge_all_groups(log, currently_converging, _my_buckets,
+        def converge_all_groups(currently_converging, _my_buckets,
                                 all_buckets, divergent_flags):
             return Effect(
-                ('converge-all', log, currently_converging, _my_buckets,
+                ('converge-all', currently_converging, _my_buckets,
                  all_buckets, divergent_flags))
 
+        def perform_ca_eff(bound_fields):
```

Comment:
it's a bit weird, but it leads to nice concise code :) it assumes that the intent passed has an `effect` attribute, which is true for all of our wrapper-intents. It works with any type of wrapper-intent and runs a performer on a SequenceDispatcher for the wrapped intent.